packages/realm-react/README.md

```diff
@@ -323,8 +323,149 @@ const SomeComponent = () => {
 }
 ```
 
+### Authentication Hooks
 
-#### Multiple Realms
+The following hooks can be used to authenticate users in your application.  They return authentication operations and a single result object which can be read to track the progress of the current result. More information about the specific auth methods can be found in the [Authenticate Users](https://www.mongodb.com/docs/realm/sdk/react-native/manage-users/authenticate-users) Documentation.
+
+## `useAuth`
+These hooks would typically be used in the `fallback` component of the `UserProvider`
+
+### `result`
+The result has the following structure:
+```tsx
+{
+  /**
+   * The current state of the operation.
+   * Enumerated by OperationState
+   */
+  state, // "not-started", "pending", "success", "error"
+
+  /**
+   * Convenience accessors, so users can write e.g. `loginResult.pending`
+   * instead of `loginResult.state === OperationState.Pending`
+   */
+  pending, // true or false
+  success, // true or false
+
+  /**
+   * The error returned from the operation, if any. This will only be populated
+   * if `state === OperationState.Error`, and will be cleared each time the
+   * operation is called.
+   */
+  error // Error based object or undefined
+}
+```
+
+This can be used to manage the state of the current login operation.
+
+### `logIn`
```

Comment:
1. As I am assuming this is all done within the `fallback` of `UserProvider`, the result of logIn would never get a chance to be used, as the application would then render as soon as `currentUser` is set on the `Realm.App`.  But perhaps for the example, it would be fine to show some sort of logging of the result.
2. Perhaps I could just wrap it in a `useEffect` that fires immediately on render, for this example.

Reply:
Yeah wrapping in `useEffect` might be suitable 👍 Do what you think makes most sense 🙂